storage/kafka/src/test/java/io/apicurio/registry/storage/util/KafkaTestContainerManager.java

```diff
@@ -0,0 +1,67 @@
+package io.apicurio.registry.storage.util;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.TimeoutException;
+
+import org.apache.kafka.clients.admin.AdminClient;
+import org.apache.kafka.clients.admin.CreateTopicsResult;
+import org.apache.kafka.clients.admin.KafkaAdminClient;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.KafkaContainer;
+
+import com.github.dockerjava.api.model.ExposedPort;
+import com.github.dockerjava.api.model.PortBinding;
+import com.github.dockerjava.api.model.Ports;
+
+import io.quarkus.test.common.QuarkusTestResourceLifecycleManager;
+
+public class KafkaTestContainerManager implements QuarkusTestResourceLifecycleManager {
```

Comment:
You could put this class -- so it's not a dup -- probably into util/kafka under test/ sources.

Reply:
> The registry API is "diying" and stops responding after some load

What if you run the tests with the streams_setup.sh script?
That's how I do it, and they worked with np so far ...

Reply:
@alesj can you work with @famartinrh to try and track down the issue of the server becoming unresponsive when using the streams storage?  A previous run appeared to have logging information indicating that connections between registry and kafka were failing... 

Here is a link to that log info:  https://bit.ly/2yuQbW4